server/app.js

```diff
@@ -5,9 +5,7 @@ const cookieParser = require("cookie-parser");
 const logger = require("morgan");
 const mongoose = require("mongoose");
 const config = require("./config.js");
```

Comment:
We should store the config files and keys in `server/config` rather than the root

Reply:
Done. Thanks Danny